geode-core/src/test/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderEventProcessorJUnitTest.java

```diff
@@ -151,4 +151,69 @@ public void validateBatchConflationWithBatchContainingDuplicateNonConflatableEve
     // Assert no events were conflated incorrectly
     assertThat(originalEvents).isEqualTo(conflatedEvents);
   }
+
+  @Test
+  public void validateBatchConflationWithBatchContainingDuplicateConflatableAndNonConflatableEvents()
+      throws Exception {
+    // This is a test for GEODE-6854.
```

Comment:
I think we might want to avoid using ticket numbers in comments if we can.  If we ever change ticketing systems, this can make it hard to track down the history of why the changes were made.  A short description of the problem is more useful IMO.

Reply:
You're right about this. I'll remove the JIRA and update the description.